ci/ci.py

```diff
@@ -388,45 +382,49 @@ def main(argv):
     branch = get_branch()
 
     # configure build flags based on the branch
+    archive_channel = None
+    editor_update_channel = None
     release_channel = None
     skip_editor_tests = False
     if branch == "master":
-        engine_channel = "stable"
-        editor_channel = "editor-alpha"
-        release_channel = "editor-stable"
+        archive_channel = "stable"
+        editor_update_channel = "editor-alpha"
+        release_channel = "stable"
         make_release = False
         engine_artifacts = args.engine_artifacts or "archived"
     elif branch == "beta":
-        engine_channel = "beta"
-        editor_channel = "beta"
-        release_channel = engine_channel
+        archive_channel = "beta"
+        editor_update_channel = "beta"
+        release_channel = "beta"
         make_release = True
         engine_artifacts = args.engine_artifacts or "archived"
     elif branch == "dev":
-        engine_channel = "alpha"
-        editor_channel = "alpha"
-        release_channel = engine_channel
+        archive_channel = "alpha"
+        editor_update_channel = "alpha"
+        release_channel = "alpha"
         make_release = True
         engine_artifacts = args.engine_artifacts or "archived"
     elif branch == "editor-dev":
-        engine_channel = None
-        editor_channel = "editor-alpha"
-        release_channel = editor_channel
+        archive_channel = "editor-alpha"
+        editor_update_channel = "editor-alpha"
+        release_channel = "editor-alpha"
         make_release = True
         engine_artifacts = args.engine_artifacts
     elif branch and branch.startswith("DEFEDIT-"):
-        engine_channel = None
-        editor_channel = None
+        archive_channel = "editor-dev"
+        editor_update_channel = "editor-dev"
+        release_channel = None
         make_release = False
         engine_artifacts = args.engine_artifacts or "archived-stable"
     else: # engine dev branch
-        engine_channel = "dev"
-        editor_channel = None
+        archive_channel = "dev"
+        editor_update_channel = "dev"
+        release_channel = None
```

Comment:
Is there a scenario where these three variables will differ?
I don't count "None" as a different name, since the usage of the channel name is controlled by other variables, e.g. "make_release".